mkt/developers/views.py

```diff
@@ -553,6 +553,7 @@ def ownership(request, addon_id, addon):
                     thread___addon=addon).delete()
 
         for author in user_form.deleted_objects:
+            author.delete()
```

Comment:
That may be sloppy. I'll have to look at that again. We don't want to delete the UserProfile object, but the object that associates a user with an admin.

The reason I believe is this:
https://docs.djangoproject.com/en/dev/releases/1.7/#miscellaneous (bullet point 2)